Frontend/src/hooks/eventListener.ts

```diff
@@ -45,6 +47,7 @@ export function useOnClickOutside<T extends Element>(
 ) {
   const ref = useRef<T>(null);
   const handleClick = useCallback(
+    // eslint-disable-next-line
     (event: any) => {
       if (ref && ref.current && ref.current.contains(event.target)) {
```

Comment:
Tror dette skal funke:
```suggestion
    (event: MouseEvent) => {
      if (event.target instanceof HTMLElement && ref.current?.contains(event.target)) {
```

Reply:
Samme med denne.